web/src/people.js

```diff
@@ -736,7 +736,9 @@ export function medium_avatar_url_for_person(person) {
         return medium_gravatar_url_for_email(person.email);
     }
 
-    return "/avatar/" + person.user_id + "/medium";
+    // We need to attach a version to the URL as a cache-breaker so that the browser
+    // will update the image in real time when user uploads a new avatar.
+    return `/avatar/${person.user_id}/medium?version=${person.avatar_version}`;
```

Comment:
I am not 100% certain that every user object this module manages has an `avatar_version`; see the code path involving `report_late_add`. So I think we probably need logic to check if `avatar_version` is set, and if not, use 1 as a fallback, at least until we can eliminate the code paths that lead to `report_late_add` being a thing.

Reply:
Yeah makes sense!